pennylane/liealg/horizontal_cartan_subalgebra.py

```diff
@@ -76,7 +76,7 @@ def horizontal_cartan_subalgebra(
     Compute a Cartan subalgebra (CSA) :math:`\mathfrak{a} \subseteq \mathfrak{m}`.
 
     A non-unique CSA is a maximal Abelian subalgebra in the horizontal subspace :math:`\mathfrak{m}` of a Cartan decomposition.
-    Note that this is sometimes called a horizontal CSA, and is different from the definition of a CSA on `Wikipedia <https://en.wikipedia.org/wiki/horizontal_cartan_subalgebra>`__.
+    Note that this is sometimes called a horizontal CSA, and is different from the definition of a CSA on `Wikipedia <https://en.wikipedia.org/wiki/Cartan_subalgebra>`__.
```

Comment:
Do we want to say "Wikipedia" or just link out to it like this?
```suggestion
    Note that this is sometimes called a horizontal CSA, and is different from the `usual definition of a CSA<https://en.wikipedia.org/wiki/Cartan_subalgebra>`__.
```

Reply:
I'm okay with it the way it is, do you have a preference?